inc/global_content/namespace.php

```diff
@@ -366,3 +370,30 @@ function prevent_site_deletion( array $caps, string $cap, int $user_id, array $a
 
 	return [ 'do_not_allow' ];
 }
+
+/**
+ * Filter all requests to the global content repo to pass-through current user cookies
+ *
+ * @param array $parsed_args Parsed request arguments.
+ * @param string $url Request URL.
+ *
+ * @filters http_request_args
+ *
+ * @return array Parsed request arguments including current session cookies.
+ */
+function filter_global_content_requests_for_auth( array $parsed_args, string $url ) : array {
+	$global_site_url = get_site_url();
+
+	if ( strpos( $url, $global_site_url ) !== 0 ) {
+		return $parsed_args;
+	}
+
+	$parsed_args['cookies'] = array_map( function( $value, $name ) {
```

Comment:
```suggestion
	$parsed_args['cookies'] = array_map( function ( $value, $name ) {
```

:D

Reply:
Surprised linter bot didn't catch that 🤔

Reply:
Was just about to say the same.